AzureResourceManager/Xray/azuredeploy_xray.parameters.json

```diff
@@ -57,7 +57,7 @@
       "value": "jdbc:postgresql://postgressrvr.postgres.database.azure.com:5432"
     },
     "db_server": {
-      "value": "postgressrvr"
+      "value": "postgressrvrr"
```

Comment:
Please use "postgressrvr". You could have an error during testing because sever with the same name existed in our subscription. I'll remove it.

Reply:
updated postgressrvr now for db_server value.